tethys_gizmos/gizmo_options/jobs_table.py

```diff
@@ -10,20 +10,189 @@
 """
 from collections import namedtuple
 import logging
+from functools import wraps
+from copy import deepcopy
 
 from tethys_portal.dependencies import vendor_static_dependencies
-from tethys_cli.cli_colors import write_warning
+from tethys_compute.models import TethysJob
```

Comment:
It's used in the global space, but I can take a look at what it would take to move everything inside functions so it can be imported at the function level.